lms/djangoapps/courseware/module_render.py

```diff
@@ -1074,7 +1075,8 @@ def handle_xblock_callback(request, course_id, usage_id, handler, suffix=None):
         return _invoke_xblock_handler(request, course_id, usage_id, handler, suffix, course=course)
 
 
-def get_module_by_usage_id(request, course_id, usage_id, disable_staff_debug_info=False, course=None):
+def get_module_by_usage_id(request, course_id, usage_id, disable_staff_debug_info=False, course=None,
+    will_recheck_access=False):
```

Comment:
I'm used to seeing the indent all the way to the first open parens. Did pylint like this?